docs/document/content/reference/api-change-history/shardingsphere-proxy/_index.en.md

```diff
@@ -4,6 +4,62 @@ weight = 1
 chapter = true
 +++
 
+## 5.0.0-beta
+
+### Data Source Configuration Item Explanation
+
+```yaml
+schemaName: # Logic schema name.
+
+dataSources: # Data sources configuration, multiple <data-source-name> available.
+  <data-source-name>: # Different from ShardingSphere-JDBC configuration, it does not need to be configured with database connection pool.
+    url: # Database URL.
+    username: # Database username.
+    password: # Database password.
+    connectionTimeoutMilliseconds: # Connection timeout milliseconds.
+    idleTimeoutMilliseconds: # Idle timeout milliseconds.
+    maxLifetimeMilliseconds: # Maximum life milliseconds.
+    maxPoolSize: 50 # Maximum connection count in the pool.
+    minPoolSize: 1  # Minimum connection count in the pool.        
+
+rules: # Keep consist with ShardingSphere-JDBC configuration.
+# ...
+```
+
+#### Authentication
+
+It is used to verify the authentication to log in ShardingSphere-Proxy, which must use correct user name and password after the configuration of them.
+
+```yaml
+rules:
+  - !AUTHORITY
+    users:
+      - root@localhost:root # <username>@<hostname>:<password>
+      - sharding@:sharding
+    provider:
+      type: NATIVE # Must be explicitly specified.
+ ```
+
+If the hostname is % or empty, it means no restrict to the user’s host.
+
+The type of the provider must be explicitly specified. Refer to [5.11 Proxy](https://shardingsphere.apache.org/document/current/en/dev-manual/proxy/) for more implementations.
```

Comment:
The link should no related to `current` version.

Reply:
> 

Done.